tests/unit/lib/test_signal_schema.py

```diff
@@ -3,17 +3,18 @@
 from typing import Optional, Union
 
 import pytest
-from sqlalchemy import Column
+import sqlalchemy as sa
```

Comment:
Why is this needed? 

I'd try to use our column `from datachain import Column` because we can eventuallly switch to a custom Column.

Reply:
Explained in comment below